src/facade/dragonfly_connection.cc

```diff
@@ -1241,6 +1243,29 @@ void Connection::HandleMigrateRequest() {
   }
 }
 
+error_code Connection::HandleRecvSocket() {
```

Comment:
the only change in this file. the code is moved to HandleRecvSocket

Reply:
+ the boilerplate in RegisterBufRings